chevah/compat/testing/assertion.py

```diff
@@ -0,0 +1,366 @@
+# -*- coding: utf-8 -*-
```

Comment:
Sice we are creating this anew (copy I know) it's a good time to place the `unicode_literals` import and revise the strings.